.gitignore

```diff
@@ -1,4 +1,5 @@
 .git/
 .idea/
 target/
-*.iml
\ No newline at end of file
+*.iml
+unit-testing.iml
```

Comment:
not necessary, because *.iml already covered your line. Clean code...

Reply:
Thank you, corrected.
Changes will be seen in the following commit.